src/cli.rs

```diff
@@ -638,8 +638,7 @@ pub(crate) fn parse_peer_info(
 	if peer_addr_str.is_none() || peer_addr_str.is_none() {
 		return Err(std::io::Error::new(
 			std::io::ErrorKind::Other,
-			"ERROR: incorrectly formatted peer
-		info. Should be formatted as: `pubkey@host:port`",
+			"ERROR: incorrectly formatted peer info. Should be formatted as: `pubkey@host:port`",
```

Comment:
FYI, If you end the line with a `\` line continuation, you can keep the indentation (e.g. https://github.com/rust-bitcoin/rust-lightning/blob/fcc0723528c12462c851d8d75f399a5c571569d2/lightning-block-sync/src/http.rs#L149). Probably doesn't matter here if the line doesn't need to be broken.